recipes/openmoltools/setup.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/setup.py b/setup.py
+index 95bf706..ca6f2e1 100644
+--- a/setup.py
++++ b/setup.py
+@@ -12,7 +12,7 @@ import subprocess
+ from distutils.ccompiler import new_compiler
+ from setuptools import setup, Extension
+ 
+-import numpy
++# import numpy
+ try:
+     from Cython.Distutils import build_ext
+     setup_kwargs = {'cmdclass': {'build_ext': build_ext}}
+@@ -51,7 +51,7 @@ setup(name='openmoltools',
+       author_email='kyleabeauchamp@gmail.com',
+       description=DOCLINES[0],
+       long_description="\n".join(DOCLINES[2:]),
+-      version=__version__,
++      version="0.8.7",
```

Comment:
why?

Reply:
It's not specified otherwise. I guess this used a preprocessor script back in the day to replace the `__version__ = VERSION` above, but that machinery (`omnia`) is not active or maintained anymore. We will end up updating `openmoltools` at some point if needed, but this package is mostly considered legacy and we need it around for a bit longer until we get rid of it.

Reply:
Anything else @isuruf?

Reply:
Can you use `os.environ["PKG_VERSION"]` to at least make sure the version matches the recipe metadata?

Reply:
Great idea, thanks!